custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js

```diff
@@ -116,10 +116,22 @@ function MapOrSectorController($scope, $compile, $location, storageService, loca
         $compile(popup[0])($scope);
     };
 
+    // watching mapData for any changes. As soon as there is change in data, height of chart is calculated and
+    // chart is refreshed.
+    $scope.$watch(function () {
+        return vm.data.mapData;
+    }, function () {
+        if (vm.data.mapData && vm.data.mapData.chart_data) {
+            var chartHeight = vm.data.mapData.chart_data[0].values.length * 70;
```

Comment:
why did this change to 70?

Reply:
There were some locations with names that were too long taking 5 lines. Earlier when it was 60, they were overlapping. So I just increased them to 70